### PR TITLE
Tag PhyloNetworks v0.5.1 [https://github.com/crsl4/PhyloNetworks.jl]

### PR DESCRIPTION
**Pull request by cecileane**
Diff vs v0.5.0: https://github.com/crsl4/PhyloNetworks.jl/compare/9898feb7fb54edb5ce72067b750f5763648950b5...ffe3c5559f30617e47c7b6f31027935535dbe0f7

We wanted to tag this version before updating the package to work with Julia v0.6.
Many thanks!

### CODE REVIEW

**Comment by tkelman** [comment]
Is the failure on Travis CI expected? https://travis-ci.org/crsl4/PhyloNetworks.jl/builds/252470835?utm_source=github_status&utm_medium=notification

**Comment by cecileane** [comment]
No, this test failure was not expected, because the tests pass just fine locally. @crsl4 and I can't reproduce the error, and there is no traceback [here](https://travis-ci.org/crsl4/PhyloNetworks.jl/builds/252470835#L539) (or [here](https://travis-ci.org/crsl4/PhyloNetworks.jl/builds/252606045#L1692) for the branch adjusted to work with julia v0.6).

The error seems to occur during a simple `print` statement, but I can't tell for sure. In any case, it occurs in a function whose purpose is to print information to a file or to the screen. I can't find a difference between the settings that we use when we run the tests locally, and the setting used by Travis CI. I would love to be able to reproduce the error and fix it.

The Travis CI build is successful when we remove the little chunk that causes the failure (last [commit](https://github.com/crsl4/PhyloNetworks.jl/commit/22e312c04c00bf8ed05da97697539beac8cfe95e) on master): see [here](https://travis-ci.org/crsl4/PhyloNetworks.jl/builds/252496222) (and [here](https://travis-ci.org/crsl4/PhyloNetworks.jl/builds/252661069) on the branch for julia v0.6).

Sorry for this very long answer...

File: PhyloNetworks/versions/0.5.1/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.5
+Compat 0.8
+StatsBase 0.9
```
Comment (tkelman) [inline PhyloNetworks/versions/0.5.1/requires]: `dof_residual` requires 0.11 https://github.com/JuliaStats/StatsBase.jl/commit/aa5851d83a5b1c5cb3969db0d3b1a6b0bc28b5c7

File: PhyloNetworks/versions/0.5.1/requires [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+DataFrames 0.7
```
Comment (tkelman) [inline PhyloNetworks/versions/0.5.1/requires]: `@formula` requires 0.9 https://github.com/JuliaStats/DataFrames.jl/commit/baa33d9614d6372d7c1e2dc96cb5d1fca71fc3e9

**Comment by cecileane** [comment]
Wow, thank you so very much Tony. I wish I had caught those requirement errors before.

I changed the requirements accordingly in the package, but not in my fork of METADATA yet. What is the best way forward?
- Is there a function to "untag" this version (like proposed in [JuliaLang/PkgDev.jl#76](https://github.com/JuliaLang/PkgDev.jl/issues/76))?
- or can I do things manually to move the tag to latest commit on master ([a5d7349](https://github.com/crsl4/PhyloNetworks.jl/commit/a5d7349d531b6fcda9610cc8a5060f49cd63447e)), and manually change both `requires` and `sha` accordingly in my fork of METADATA, then adjust this pull request?
- or change `requires` manually in METADATA, but let the tag point to a version of the package with errors in its REQUIRE file?

**Comment by tkelman** [comment]
You can do either the second or third bullets. The METADATA requires takes priority and doesn't have to exactly match the package REQUIRE file for tagged versions, the METADATA version can be amended after the fact. For the first bullet, check out https://github.com/attobot/attobot which automates the parts of the process that PkgDev previously did.

**Comment by cecileane** [comment]
I used [AttoBot](https://github.com/attobot/attobot) (thank you!), but it created a new PR #10240 instead of updating this one...

**Comment by tkelman** [comment]
it can update its own pr's, but not existing pkgdev style ones from personal forks. will close this in favor of that one

**Pull request closed without merging**